libs/recipes/hdf5/targets.mk

```
@@ -0,0 +1,3 @@
+HDF5_WASM_LIB = $(WASM)/lib/libhdf5.a
+OPTIONAL_HDF5_LIBS += $(HDF5_WASM_LIB)
+OPTIONAL_HDF5_BINS += h5cc
```

```suggestion
OPTIONAL_WASM_LIBS += $(HDF5_WASM_LIB)
OPTIONAL_WASM_BINS += h5cc
```

These targets should be named as above, so that they're automatically included when running `make all`.